src/vogsphere.html

```diff
@@ -15,15 +15,33 @@ <h3>Basics</h3>
             <input id="vogsphere__character-name" name="characterName" type="text" required>
         </div>
 
-        <div class="__input-group">
-            <label for="vogsphere__member-group">Member Group</label>
-            <select id="vogsphere__member-group" name="memberGroup" required>
-                <option value="">--Please choose an option--</option>
-                <option value="crew">Crew</option>
-                <option value="scientist">Scientist</option>
-                <option value="passenger">Passenger</option>
-            </select>
-        </div>
+        <!--Added radio buttons for member-group selection-->
+        <fieldset>
+            <legend>
+                Member Group
+            </legend>
+            <!--Selected class minimal to continue same format below-->
```

Comment:
While this comment is helpful in the context of this PR, I don't think it's needed long-term, since the CSS is all meant to be replaced. Could you please remove this comment?